UnityProject/Assets/Scripts/Clothing/BeltData.cs

```diff
@@ -0,0 +1,63 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "ContainerData", menuName = "ScriptableObjects/BeltData", order = 1)]
+public class BeltData : BaseClothData
+{
+	public EquippedData sprites;
+	public ItemStorageStructure structure;
+	public ItemStorageCapacity capacity;
+	public ItemStoragePopulator populator;
+
+	public override Sprite SpawnerIcon()
+	{
+		return sprites.ItemIcon.Sprites[0];
```

Comment:
getting NRE here when trying to find it in dev spawner:
```
IndexOutOfRangeException: Index was outside the bounds of the array.
BeltData.SpawnerIcon () (at Assets/Scripts/Clothing/BeltData.cs:14)
DevSpawnerListItemController.Initialize (Lucene.Net.Documents.Document resultDoc) (at Assets/Scripts/UI/DevTools/DevSpawnerListItemController.cs:68)
GUI_DevSpawner.CreateListItem (Lucene.Net.Documents.Document doc) (at Assets/Scripts/UI/DevTools/GUI_DevSpawner.cs:144)
GUI_DevSpawner.Search () (at Assets/Scripts/UI/DevTools/GUI_DevSpawner.cs:118)
UnityEngine.Events.InvokableCall.Invoke () (at /Users/builduser/buildslave/unity/build/Runtime/Export/UnityEvent/UnityEvent.cs:166)
UnityEngine.Events.UnityEvent`1[T0].Invoke (T0 arg0) (at /Users/builduser/buildslave/unity/build/Runtime/Export/UnityEvent/UnityEvent/UnityEvent_1.cs:63)
UnityEngine.UI.InputField.SendOnSubmit () (at /Applications/Unity/Hub/Editor/2019.2.9f1/Unity.app/Contents/Resources/PackageManager/BuiltInPackages/com.unity.ugui/Runtime/UI/Core/InputField.cs:2171)
UnityEngine.UI.InputField.DeactivateInputField () (at /Applications/Unity/Hub/Editor/2019.2.9f1/Unity.app/Contents/Resources/PackageManager/BuiltInPackages/com.unity.ugui/Runtime/UI/Core/InputField.cs:2970)
UnityEngine.UI.InputField.OnUpdateSelected (UnityEngine.EventSystems.BaseEventData eventData) (at /Applications/Unity/Hub/Editor/2019.2.9f1/Unity.app/Contents/Resources/PackageManager/BuiltInPackages/com.unity.ugui/Runtime/UI/Core/InputField.cs:1827)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IUpdateSelectedHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at /Applications/Unity/Hub/Editor/2019.2.9f1/Unity.app/Contents/Resources/PackageManager/BuiltInPackages/com.unity.ugui/Runtime/EventSystem/ExecuteEvents.cs:99)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at /Applications/Unity/Hub/Editor/2019.2.9f1/Unity.app/Contents/Resources/PackageManager/BuiltInPackages/com.unity.ugui/Runtime/EventSystem/ExecuteEvents.cs:261)
UnityEngine.EventSystems.EventSystem:Update() (at /Applications/Unity/Hub/Editor/2019.2.9f1/Unity.app/Contents/Resources/PackageManager/BuiltInPackages/com.unity.ugui/Runtime/EventSystem/EventSystem.cs:377)

```

Reply:
Ah- it's because BeltData isn't supposed to be spawnable, it's just the base prefab for all belts. The parent belt is just... belt so we can always move that data into here to prevent this

Reply:
or wait, rereading this now... which one were you trying to spawn?